python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -560,9 +561,11 @@ def run_through_rpc(
             raise AttributeError(
                 "Please make sure USE_RANDOM is ON in the config.cmake " "on the remote devices"
             )
-        args = [nd.empty(x[0], dtype=x[1], ctx=ctx) for x in build_result.arg_info]
-        for arg in args:
-            random_fill(arg)
+        args = [nd.array(np.zeros(x[0], dtype=x[1]), ctx=ctx) for x in build_result.arg_info]
```

Comment:
This solution maybe have potential problem in the future. If we import a model from other frontend(like tf), if our compiler could leverage the model layer name and become into compiler func name so that we could debug (profile) / partition subgraph more easily, maye we couldn’t see ‘scatter' any more.

Reply:
Here, I look for the task name coming from https://github.com/apache/tvm/blob/7b1a6ea704c4aa153b8234a92720edd637500def/python/tvm/relay/op/strategy/cuda.py#L787

Since this name is always associated with a tuning task, I don't see this could be a problem.

Reply:
Ok. If so, there is no problem. Thanks for the explainment.